resources/openshift-haproxy-router/alerts.yaml

```diff
@@ -1,79 +1,78 @@
 apiVersion: v1
 kind: Alert
-app: OpenShift HAProxy Router
+app: 'OpenShift HAProxy Router'
 version: 1.0.0
 appVersion:
 - '3.11'
-- '4.3'
+- '4.7'
 descriptionFile: ALERTS.md
 configurations:
 - kind: Prometheus
   data: |
     groups:
     - name: OpenShift-HAProxy-Router
       rules:
-      - alert: RouterDown
-        expr: |
-          absent((count(haproxy_process_start_time_seconds) < 1))
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: Router HAProxy down. No instances running.
-      - alert: DownTimeInService
-        expr: |
-          haproxy_backend_downtime_seconds_total > 0
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: DownTime detected in service. Route {{$labels.route}}, pod {{labels.pod}}
-      - alert: RouteDown
-        expr: |
-          sum by (route) (haproxy_server_up==1) == 0
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: All servers are down in route {{$labels.route}}
-      - alert: HighLatency
-        expr: |
-          max by (route)(haproxy_server_http_average_response_latency_milliseconds{route!=""}) > 250
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: High latency in at least one server for the route {{$labels.route}}
-      - alert: PodHealthCheckFailure
-        expr: |
-          rate(haproxy_server_check_failures_total[5m]) > 0
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: Recurrent health check failure in pod {{$labels.pod}} and route {{$labels.route}}
-      - alert: QueueNotEmptyInRoute
-        expr: |
-          sum by (route)(haproxy_server_current_queue{route!=""}) > 0
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: Queue not empty in route {{$labels.route}}
-      - alert: HighErrorRateInRoute
-        expr: |
-          sum by (route) (rate(haproxy_server_http_responses_total{code!="2xx"}[5m])) /
-            sum  by (route) (rate(haproxy_server_http_responses_total{}[5m]))
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: High error rate in route {{$labels.route}}
-      - alert: ConnectionErrorsInRoute
-        expr: |
-          sum by (route)(rate(haproxy_server_connection_errors_total{route!=""}[5m])) > 0
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: Recurring connection errors in route {{$labels.route}}
+        - alert: '[OpenShift-HAProxy-Router] Router Down'
+          expr: |
+            absent((count(haproxy_process_start_time_seconds) < 1))
```

Comment:
I think the 'absent' alerts must end with a '== 1'

Reply:
Fixed